common/generate_bundle_base

```diff
@@ -50,7 +50,7 @@ fi
 
 if [ -n "${MASTER_OPTS[MODEL_CONSTRAINTS]}" ]; then
     juju set-model-constraints ${MASTER_OPTS[MODEL_CONSTRAINTS]}
-else
+elif ! ${MASTER_OPTS[HYPERCONVERGED_DEPLOYMENT]}; then
```

Comment:
nit: not sure if the check should use "" in case the option is empty like above?

elif [ -n "${MASTER_OPTS[HYPERCONVERGED_DEPLOYMENT]}" ]; then

Reply:
HYPERCONVERGED_DEPLOYMENT defaults to false (bool) and is only ever set to true otherwise

Reply:
LGTM